src/helper.js

```diff
@@ -138,6 +138,28 @@ const escapeRegex = (str) => {
   return str.replace(/[.*+\-?^${}()|[\]\\]/g, '\\$&')
 }
 
+/**
+ * @param {string} line
+ * @return {number}
+ */
+const parseLogDate = (line) => {
+  const logDate = Date.parse(line.split(' ')[0])
+
+  if (Number.isNaN(logDate)) throw new Error('the log line don\'t have a timestamp')
```

Comment:
can we make error message in snake uppercase format similar to others? e.g. ERR_DATE_TS_INVALID